drivers/storage/class/ramdisk/ramdisk.c

```diff
@@ -1260,21 +1260,69 @@ RamdiskDeviceControl(IN PDEVICE_OBJECT DeviceObject,
                     goto CallWorker;
                 }
 
-                /* We're done */
+                /* We are done */
                 Information = Irp->IoStatus.Information;
                 break;
             }
 
-            case IOCTL_DISK_GET_DRIVE_LAYOUT:
             case IOCTL_DISK_GET_LENGTH_INFO:
+            {
+                PGET_LENGTH_INFORMATION gli = Irp->AssociatedIrp.SystemBuffer;
```

Comment:
Not a big deal, but for future reference, our kernel-mode code generally prefers unabbreviated CamelCase names, e.g. `LengthInformation` or `LengthInfo`, `GptInformation` or `GptAttributes`.

Reply:
changed them. i wanna make this pull as good as possible